modules/apps/updater.nix

```diff
@@ -25,24 +25,36 @@ in
         description = "URL for OTA updates";
         apply = x: if lib.hasSuffix "/" x then x else x + "/";
       };
+
+      flavor = mkOption {
+        type = types.nullOr types.str;
```

Comment:
How about:
`types.strMatching "(grapheneos|lineageos)"`
with the default being "grapheneos"?
What you've currently been calling the "vanilla" updater flavor should probably be the "grapheneos" updater flavor, since the updater app was written for that project.

Reply:
Ah, good to know. I'd rather go with an enum, than strMatching, though.